docs/content/build-instructions.md

```diff
@@ -192,3 +192,19 @@ In addition, `pkgconfig` can be configured to find lcm.pc:
 ```shell
 export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:$LCM_LIBRARY_DIR/pkgconfig
 ```
+
+## Bazel
```

Comment:
Do you think it would be helpful to document the commands a user will need to run to build with Bazel?

Reply:
Yup, I added two more paragraphs about it.